CsMerge/CsMerge.cs

```diff
@@ -123,49 +128,65 @@ public class CsMerge {
         XDocument theirDocument = XDocument.Parse( theirContent );
         XDocument baseDocument = XDocument.Parse( baseContent );
 
-        var projFileName = Path.GetFileName( conflict );
+        var resolved = false;
+
+        try {
```

Comment:
If an exception is thrown, catch it and log it, and then allow the user to merge using the merge tool if they desire.